spec/spacex/launches_spec.rb

```diff
@@ -208,6 +208,91 @@
     end
   end
 
+  context "#info('68')", vcr: { cassette_name: 'launches/68' } do
+    subject do
+      SPACEX::Launches.info('68')
+    end
+
+    it 'returns a specific launch' do
+      expect(subject.flight_number).to eq 68
+      expect(subject.mission_name).to eq 'Telstar 18V'
+      expect(subject.mission_id.first).to eq 'F4F83DE'
+      expect(subject.launch_year).to eq '2018'
+      expect(subject.launch_date_unix).to eq 1_536_554_700
+      expect(subject.launch_date_utc).to eq '2018-09-10T04:45:00.000Z'
+      expect(subject.launch_date_local).to eq '2018-09-10T00:45:00-04:00'
+      expect(subject.is_tentative).to eq false
+      expect(subject.tentative_max_precision).to eq 'hour'
+      expect(subject.rocket.rocket_id).to eq 'falcon9'
+      expect(subject.rocket.rocket_name).to eq 'Falcon 9'
+      expect(subject.rocket.rocket_type).to eq 'FT'
+      expect(subject.rocket.first_stage.cores.first.core_serial).to eq 'B1049'
+      expect(subject.rocket.first_stage.cores.first.flight).to eq 1
```

Comment:
This works but in other tests we have been simply comparing the entire hash instead of verifying each key value pair.